commit b19b9137b9d6f0a57c3dbb86992be8ef2d1d1526
Author: Albert Astals Cid <aacid@kde.org>
Date:   Thu Jul 4 19:40:05 2019 +0200

    Remove ifdef for Qt 5.11 since we require that version now

diff --git src/kstatusbarjobtracker.cpp src/kstatusbarjobtracker.cpp
index e548728..b4e109e 100644
--- src/kstatusbarjobtracker.cpp
+++ src/kstatusbarjobtracker.cpp
@@ -149,7 +149,11 @@ void KStatusBarJobTracker::Private::ProgressWidget::killJob()
 void KStatusBarJobTracker::Private::ProgressWidget::init(KJob *job, QWidget *parent)
 {
     widget = new QWidget(parent);
+#if QT_VERSION < QT_VERSION_CHECK(5, 11, 0)
+    int w = fontMetrics().width(QStringLiteral(" 999.9 kB/s 00:00:01 ")) + 8;
+#else
     int w = fontMetrics().horizontalAdvance(QStringLiteral(" 999.9 kB/s 00:00:01 ")) + 8;
+#endif
     box = new QHBoxLayout(widget);
     box->setContentsMargins(0, 0, 0, 0);
     box->setSpacing(0);
